src/libraries/Native/Unix/configure.cmake

```diff
@@ -442,37 +444,69 @@ if (CLR_CMAKE_TARGET_LINUX)
     set(HAVE_SUPPORT_FOR_DUAL_MODE_IPV4_PACKET_INFO 1)
 endif ()
 
-check_c_source_runs(
-    "
-    #include <stdlib.h>
-    #include <time.h>
-    #include <sys/time.h>
-    int main(void)
-    {
-        int ret;
-        struct timespec ts;
-        ret = clock_gettime(CLOCK_MONOTONIC, &ts);
-        exit(ret);
-        return 0;
-    }
-    "
-    HAVE_CLOCK_MONOTONIC)
-
-check_c_source_runs(
-    "
-    #include <stdlib.h>
-    #include <time.h>
-    #include <sys/time.h>
-    int main(void)
-    {
-        int ret;
-        struct timespec ts;
-        ret = clock_gettime(CLOCK_REALTIME, &ts);
-        exit(ret);
-        return 0;
-    }
-    "
-    HAVE_CLOCK_REALTIME)
+if(CLR_CMAKE_TARGET_IOS)
+    # Manually set results from check_c_source_runs() since it's not possible to actually run it during CMake configure checking
+    unset(HAVE_SHM_OPEN_THAT_WORKS_WELL_ENOUGH_WITH_MMAP)
+    unset(HAVE_CLOCK_MONOTONIC) # only exists on iOS 10+
+    unset(HAVE_CLOCK_REALTIME)  # only exists on iOS 10+
```

Comment:
What's the minimum target iOS?  9 stopped getting patches 3 years ago (modulo one patch for GPS rollover last year).  Even 10 looks to be out of support other than the GPS fix.

Reply:
The minimum target that Xamarin supports today is iOS 7 for devices and iOS 8 for the simulator so I replicated that here. We're discussing with PM but it looks like we won't be able to meaningfully bump the minimum version, definitely not to 10.